packages/x-components/src/x-modules/queries-preview/components/query-preview.vue

```diff
@@ -223,12 +235,16 @@
         }
       );
 
-      const previewItemQuery = this.queryPreviewInfo.query;
+      const previewItemQuery = this.queryOfQueryPreview;
```

Comment:
I think we can skip the `previewItemQuery` variable and just use `this.queryOfQueryPreview`